mmv1/third_party/terraform/website/docs/r/sql_database_instance.html.markdown

```diff
@@ -572,6 +579,36 @@ performing filtering in a Terraform config.
 
 * `server_ca_cert.0.sha1_fingerprint` - SHA Fingerprint of the CA Cert.
 
+## Switchover (SQL Server Only)
+Users can perform a switchover on any direct `cascadable` replica by following the steps below.
+
+  ~>**WARNING:** Failure to follow these steps can lead to data loss (You will be warned during plan stage). To prevent data loss during a switchover, please verify your plan with the checklist below.
+
+### Steps to Invoke Switchover
```

Comment:
It may be useful to include or point to configs so that users can see the changes they need to make